pkg/cmd/context/create/create.go

```diff
@@ -0,0 +1,102 @@
+package create
+
+import (
+	"context"
+
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/kafka/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/icon"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/iostreams"
+	"github.com/redhat-developer/app-services-cli/pkg/core/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/core/logging"
+	"github.com/redhat-developer/app-services-cli/pkg/core/profile"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/profileutil"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	IO         *iostreams.IOStreams
+	Logger     logging.Logger
+	Connection factory.ConnectionFunc
+	localizer  localize.Localizer
+	Context    context.Context
+	Profiles   profile.IContext
+
+	name       string
+	kafkaID    string
+	registryID string
+}
+
+// NewCreateCommand creates a new command to create contexts
+func NewCreateCommand(f *factory.Factory) *cobra.Command {
+
+	opts := &options{
+		Connection: f.Connection,
+		IO:         f.IOStreams,
+		Logger:     f.Logger,
+		localizer:  f.Localizer,
+		Profiles:   f.Profile,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "create",
+		Short:   f.Localizer.MustLocalize("context.create.cmd.shortDescription"),
+		Long:    f.Localizer.MustLocalize("context.create.cmd.longDescription"),
+		Example: f.Localizer.MustLocalize("context.create.cmd.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runCreate(opts)
+		},
+	}
+
+	flags := flagutil.NewFlagSet(cmd, opts.localizer)
+
+	flags.StringVar(&opts.name, "name", "", opts.localizer.MustLocalize("context.common.flag.name"))
+	flags.StringVar(&opts.kafkaID, "kafka-id", "", opts.localizer.MustLocalize("context.common.flag.name"))
+	flags.StringVar(&opts.registryID, "registry-id", "", opts.localizer.MustLocalize("context.common.flag.name"))
```

Comment:
Would those be temporary until we have `kafka use` etc. support?

Reply:
I think we should give user ability to set service identifiers while creating context?

Reply:
Sounds good